packages/tokamak/contracts-bedrock/foundry.toml

```diff
@@ -24,6 +26,7 @@ bytecode_hash = 'none'
 build_info = true
 build_info_path = 'artifacts/build-info'
 
+
```

Comment:
Thank you!
I think we can remove this empty line!

Reply:
@rlgns98kr 
Thank you! i removed empty lines : )